guildwars2/guild/general.py

```diff
@@ -213,9 +301,51 @@ class GeneralGuild:
                         inline=False)
                     counter += 1
         if counter == 0:
-            return await ctx.send(
-                "No stash log entries yet for {}".format(guild_name.title()))
+            return await ctx.send("No stash log entries yet for {}".format(
+                guild_name.title()))
         try:
             await ctx.send(embed=data)
         except discord.Forbidden:
             await ctx.send("Need permission to embed links")
+
+    @guild.command(name="set")
+    @commands.cooldown(1, 10, BucketType.user)
+    async def guild_set(self, ctx, *, guild_name: str):
```

Comment:
"set" is not very descriptive command name i think. Probably "default" would work better, then you can have in one command.
`guild_name=None` to clear the default, and get rid of the unset command.

Reply:
Removed unset command and merged it into new default